# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/transform-3d/reftest.list
# Commit: e24ed89246a4
# Full Hash: e24ed89246a4a6c6b4a567ec412f0e3295b4570d
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-26 09:46:09
# Regressor Bug: 1766017
# File Overlap Count: 1
# Description:
#   Bug 1766017 - Do a better job of clipping off-screen surface allocation sizes r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D144534
# ==============================================================================

diff -r cdc42b7822f1 -r e24ed89246a4 layout/reftests/transform-3d/reftest.list
--- a/layout/reftests/transform-3d/reftest.list	Mon Apr 25 22:28:35 2022 +0000
+++ b/layout/reftests/transform-3d/reftest.list	Mon Apr 25 23:40:40 2022 +0000
@@ -94,7 +94,7 @@
 == intermediate-1.html intermediate-1-ref.html
 == preserves3d-nested-filter-1.html preserves3d-nested-filter-1-ref.html
 != preserve3d-scale.html about:blank
-fuzzy(0-50,0-1600) == preserve3d-scale.html preserve3d-scale-ref.html
+fuzzy(0-52,0-1600) == preserve3d-scale.html preserve3d-scale-ref.html
 fuzzy(0-1,0-5) == perspective-overflow-1.html perspective-overflow-1-ref.html
 == perspective-overflow-2.html perspective-overflow-2-ref.html
 == 1544995-1.html 1544995-1-ref.html
